projects/js-packages/connection/components/connection-status-card/index.jsx

```diff
@@ -41,11 +33,14 @@ const ConnectionStatusCard = props => {
 		title,
 		connectionInfoText,
 		onDisconnected,
+		connectedPlugins,
+		currentPlugin,
```

Comment:
There's a better way to fetch the current plugin now: https://github.com/Automattic/jetpack/tree/master/projects/js-packages/config

It would be awesome if we used that. See an example here: https://github.com/Automattic/jetpack/pull/21672

Reply:
That's so cool! I have updated the `ConnectionStatusCard` to use this new method and will change the webpack config files in the dependent PRs for the plugins to include the `consumer_slug` value.

Reply:
https://github.com/Automattic/jetpack/pull/21672 already adds it for Backup and Jetpack. I'll try to get it merged

Reply:
Ah OK - I worked the webpack changes into #21421 and #20914 already 😅 . If #21672 gets merged first we can re-work those sub-branches.